init.zsh

```diff
@@ -7,8 +7,12 @@ if [[ -z "$ANTIGEN_HS_MY" ]]; then
     ANTIGEN_HS_MY="$ANTIGEN_HS_HOME/../MyAntigen.hs"
 fi
 
+antigen-hs-init () {
+  cd "$ANTIGEN_HS_HOME" && stack build
+}
+
 antigen-hs-compile () {
-  runghc -i"$ANTIGEN_HS_HOME/" -- "$ANTIGEN_HS_MY"
+  stack exec -- runghc -i"$ANTIGEN_HS_HOME/" -- "$ANTIGEN_HS_MY"
```

Comment:
Won't users require `stack` then? People who don't know Haskell should be able to use this. I'm fine with this as long as users can in a normal setup (say Ubuntu 14.04) do `sudo apt-get install stack` or something.